hw/top_earlgrey/ip_autogen/rv_plic/doc/checklist.md

```diff
@@ -53,23 +53,23 @@ Code Quality  | [CDC_SYNCMACRO][]       | N/A         |
-[MISSING_FUNC]:        ../../../../doc/project_governance/checklist/README.md#missing_func
-[FEATURE_FROZEN]:      ../../../../doc/project_governance/checklist/README.md#feature_frozen
-[FEATURE_COMPLETE]:    ../../../../doc/project_governance/checklist/README.md#feature_complete
-[AREA_CHECK]:          ../../../../doc/project_governance/checklist/README.md#area_check
-[PORT_FROZEN]:         ../../../../doc/project_governance/checklist/README.md#port_frozen
-[ARCHITECTURE_FROZEN]: ../../../../doc/project_governance/checklist/README.md#architecture_frozen
-[REVIEW_TODO]:         ../../../../doc/project_governance/checklist/README.md#review_todo
-[STYLE_X]:             ../../../../doc/project_governance/checklist/README.md#style_x
-[LINT_PASS]:           ../../../../doc/project_governance/checklist/README.md#lint_pass
-[CDC_SETUP]:           ../../../../doc/project_governance/checklist/README.md#cdc_setup
-[FPGA_TIMING]:         ../../../../doc/project_governance/checklist/README.md#fpga_timing
-[CDC_SYNCMACRO]:       ../../../../doc/project_governance/checklist/README.md#cdc_syncmacro
-[SEC_CM_DOCUMENTED]:   ../../../../doc/project_governance/checklist/README.md#sec_cm_documented
-[SEC_RND_CNST]:        ../../../../doc/project_governance/checklist/README.md#sec_rnd_cnst
+[NEW_FEATURES]:        ../../../../../doc/project_governance/checklist/README.md#new_features
+[BLOCK_DIAGRAM]:       ../../../../../doc/project_governance/checklist/README.md#block_diagram
+[DOC_INTERFACE]:       ../../../../../doc/project_governance/checklist/README.md#doc_interface
+[MISSING_FUNC]:        ../../../../../doc/project_governance/checklist/README.md#missing_func
+[FEATURE_FROZEN]:      ../../../../../doc/project_governance/checklist/README.md#feature_frozen
+[FEATURE_COMPLETE]:    ../../../../../doc/project_governance/checklist/README.md#feature_complete
+[AREA_CHECK]:          ../../../../../doc/project_governance/checklist/README.md#area_check
+[PORT_FROZEN]:         ../../../../../doc/project_governance/checklist/README.md#port_frozen
+[ARCHITECTURE_FROZEN]: ../../../../../doc/project_governance/checklist/README.md#architecture_frozen
+[REVIEW_TODO]:         ../../../../../doc/project_governance/checklist/README.md#review_todo
+[STYLE_X]:             ../../../../../doc/project_governance/checklist/README.md#style_x
+[LINT_PASS]:           ../../../../../doc/project_governance/checklist/README.md#lint_pass
+[CDC_SETUP]:           ../../../../../doc/project_governance/checklist/README.md#cdc_setup
+[FPGA_TIMING]:         ../../../../../doc/project_governance/checklist/README.md#fpga_timing
+[CDC_SYNCMACRO]:       ../../../../../doc/project_governance/checklist/README.md#cdc_syncmacro
+[SEC_CM_DOCUMENTED]:   ../../../../../doc/project_governance/checklist/README.md#sec_cm_documented
+[SEC_RND_CNST]:        ../../../../../doc/project_governance/checklist/README.md#sec_rnd_cnst
```

Comment:
I don't see anything like this for `D2`, remove?
```suggestion
```

Reply:
Looks like it was moved to D2S (and renamed) in commit `f28ef61435fd41b546acc1f200371eda4f555032`.